pkg/controller/dependences.go

```diff
@@ -316,7 +316,7 @@ func newDependencies(
 		PodLister:                   kubeInformerFactory.Core().V1().Pods().Lister(),
 		NodeLister:                  nodeLister,
 		SecretLister:                kubeInformerFactory.Core().V1().Secrets().Lister(),
-		ConfigMapLister:             labelFilterKubeInformerFactory.Core().V1().ConfigMaps().Lister(),
+		ConfigMapLister:             kubeInformerFactory.Core().V1().ConfigMaps().Lister(),
```

Comment:
Cause function `NewFakeDependencies()` mixes kubeInformerFactory and labelFilterKubeInformerFactory in `deps` and `deps.Controls` which makes some tests errors.  https://github.com/pingcap/tidb-operator/blob/48ae8298f491c0f7c57ef53041912cffcb5ce6a6/pkg/controller/dependences.go#L422-L423

Reply:
Cause function `NewFakeDependencies()` mixes kubeInformerFactory and labelFilterKubeInformerFactory in `deps` and `deps.Controls` which makes some tests errors.  https://github.com/pingcap/tidb-operator/blob/48ae8298f491c0f7c57ef53041912cffcb5ce6a6/pkg/controller/dependences.go#L422-L423

Reply:
you should be ok to use `labelFilterKubeInformerFactory` same as `kubeInformerFactory`. could you fix the test?
the origin object to use `labelFilterKubeInformerFactory` for ConfigMap is to avoid saving all ConfigMap which it's said it may be to huge.

Reply:
OK, fixed some tests cases.